<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QWidget" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>999</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="parameter_tab">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>-1</y>
     <width>1101</width>
     <height>1001</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="plot_tab">
    <attribute name="title">
     <string>Plot</string>
    </attribute>
    <widget class="QListWidget" name="observation_widget">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>0</y>
       <width>331</width>
       <height>261</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
    <widget class="QTableWidgetEnhancedDrop" name="parameter_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>761</width>
       <height>261</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>851</width>
       <height>631</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="plot_layout"/>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>900</y>
       <width>851</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="toolbar_layout">
      <property name="rightMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <widget class="QPushButton" name="update_plot_button">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>270</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>UpdatePlot</string>
     </property>
    </widget>
    <zorder>parameter_table</zorder>
    <zorder>observation_widget</zorder>
    <zorder>gridLayoutWidget</zorder>
    <zorder>gridLayoutWidget_2</zorder>
    <zorder>update_plot_button</zorder>
   </widget>
   <widget class="QWidget" name="fit_tab">
    <attribute name="title">
     <string>Fit</string>
    </attribute>
    <widget class="QTableWidgetEnhanced" name="fit_select_table">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>421</width>
       <height>261</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QTableWidgetEnhanced" name="fit_result_table">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>0</y>
       <width>661</width>
       <height>261</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>900</y>
       <width>851</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="toolbar_fit_layout">
      <property name="rightMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>851</width>
       <height>631</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="plot_fit_layout"/>
    </widget>
    <widget class="QPushButton" name="update_fit_result_button">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>270</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>UpdateFit</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="info">
   <property name="geometry">
    <rect>
     <x>873</x>
     <y>360</y>
     <width>231</width>
     <height>531</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="open_code_button">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>902</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Measurement Code</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_explorer_button">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>950</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Explorer Window</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTableWidgetEnhanced</class>
   <extends>QTableWidget</extends>
   <header>qutip_enhanced/qtgui/custom_widgets.h</header>
  </customwidget>
  <customwidget>
   <class>QTableWidgetEnhancedDrop</class>
   <extends>QTableWidget</extends>
   <header>qutip_enhanced/qtgui/custom_widgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
